src/renderer/views/Playlist/Playlist.js

```diff
@@ -152,16 +163,56 @@ export default defineComponent({
         this.isLoading = false
       }).catch((err) => {
         console.error(err)
-        if (process.env.IS_ELECTRON && this.backendPreference === 'invidious' && this.backendFallback) {
-          console.warn('Error getting data with Invidious, falling back to local backend')
-          this.getPlaylistLocal()
+        if (this.backendPreference === 'invidious' && this.backendFallback) {
+          if (process.env.IS_ELECTRON && this.fallbackPreference === 'local') {
+            console.warn('Error getting data with Invidious, falling back to local backend')
+            this.getPlaylistLocal()
+          } else if (this.fallbackPreference === 'piped') {
+            console.warn('Error getting data with Invidious, falling back to Piped backend')
+            this.getPlaylistPiped()
+          } else {
+            this.isLoading = false
+          }
         } else {
           this.isLoading = false
           // TODO: Show toast with error message
         }
       })
     },
 
+    getPlaylistPiped: async function () {
+      try {
+        this.isLoading = true
+        const { playlist, videos, nextpage } = await getPipedPlaylist(this.playlistId)
+        this.infoData = playlist
+        this.continuationData = nextpage
+        this.playlistItems = this.playlistItems.concat(videos)
+
+        this.updateSubscriptionDetails({
+          channelThumbnailUrl: pipedImageToYouTube(playlist.channelThumbnail),
+          channelName: playlist.channelName,
+          channelId: playlist.channelId
+        })
+        this.isLoading = false
+      } catch (err) {
+        console.error(err)
+        if (this.backendPreference === 'invidious' && this.backendFallback) {
+          if (process.env.IS_ELECTRON && this.fallbackPreference === 'local') {
+            console.warn('Error getting data with Piped, falling back to local backend')
+            this.getPlaylistLocal()
+          } else if (this.fallbackPreference === 'invidious') {
+            console.warn('Error getting data with Piped, falling back to Invidious backend')
+            this.getPlaylistPiped()
```

Comment:
Unless I am reading this wrong...
```suggestion
            this.getPlaylistInvidious()
```